src/commands/email/send.js

```diff
@@ -0,0 +1,140 @@
+const { flags } = require('@oclif/command');
+const { BaseCommand } = require('@twilio/cli-core').baseCommands;
+const sgMail = require('@sendgrid/mail');
+
+class send extends BaseCommand {
+  async run() {
+    await super.run();
+    this.loadArguments();
+    if (!process.env.SENDGRID_API_KEY) {
+      this.logger.error('Make sure you have an environmental variable called SENDGRID_API_KEY set up with your SendGrid API key. Visit https://app.sendgrid.com/settings/api_keys to get an API key.');
+      return this.exit(1);
+    }
+    this.fromEmail = await this.promptForFromEmail();
+    const validFromEmail = this.validateEmail(this.fromEmail);
+    const stringFromEmail = validFromEmail.toString();
+    await this.promptForToEmail();
+    const validToEmail = this.validateEmail(this.toEmail);
+    this.subjectLine = await this.promptForSubject();
+    await this.promptForText();
+    const sendInfomation = { to: validToEmail, from: stringFromEmail, subject: this.subjectLine, text: this.emailText, html: '<p>' + this.emailText + '</p>' };
+    await this.sendEmail(sendInfomation);
+  }
+
+  loadArguments() {
+    this.toEmail = this.flags.toEmail;
+    this.fromEmail = this.flags.fromEmail;
+    this.subjectLine = this.flags.subjectLine;
+    this.emailText = this.flags.emailText;
+  }
+
+  validateEmail(email) {
+    var emailList = [];
+    var validEmail;
+    const multipleEmail = email.includes(',');
+    if (multipleEmail === true) {
+      emailList = email.split(',').map(item => {
+        return item.trim();
+      });
+    } else {
+      emailList[0] = email;
+    }
+    emailList.forEach(emailAddress => {
+      const emailVerdict = emailAddress.includes('@');
+      if (emailVerdict === false) {
+        this.logger.error(emailAddress + ' is not a valid email.');
+        validEmail = false;
+      }
+    });
+    if (validEmail === false) {
+      this.logger.error('Email could not be sent please re-run the command with valid email addresses.');
```

Comment:
Email could not be sent please re-run the command with valid email addresses.

should be:

Email could not be sent, please re-run the command with valid email addresses.